crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -47,7 +47,9 @@ def restore_chef
               if proposal?(filename) && type == :databags
                 bc_name, prop = filename.split("-")
                 prop.gsub!(/.json$/, "")
-                Proposal.create(barclamp: bc_name, name: prop, properties: record.raw_data)
+                proposal = Proposal.where(barclamp: bc_name, name: prop).first_or_initialize
```

Comment:
right, sorry next time i will test it before. I thought Rails would be so smart to know what I want... will fix